export/wiki/stage_species.py

```diff
@@ -207,7 +207,8 @@ def extract(self, proxy: UEProxyStructure) -> Any:
         if is_creature_tameable(species, variants, overrides):
             out.flags.append('isTameable')
 
-        out.levelCaps = convert_level_data(species, dcsc)
+        if not species.bIsBossDino[0]:
```

Comment:
I preferred this test being in the function, although it could return a default `LevelData` or `None` and be or-ed with a default value. Your choice.